.github/workflows/release.yml

```diff
@@ -24,7 +24,7 @@ jobs:
               run: npm i
 
             - name: Build
-              run: npm run build
```

Comment:
We don't need to run the build to publish the package?

Reply:
there is a `precompile` script